ludwig/features/binary_feature.py

```diff
@@ -212,14 +215,15 @@ def _setup_metrics(self):
     def update_metrics(self, targets, predictions):
         for metric, metric_fn in self.metric_functions.items():
             if metric == LOSS:
-                metric_fn.update_state(
-                    targets,
-                    predictions,
-                    positive_class_weight=self.loss['positive_class_weight'],
-                    robust_lambda=self.loss['robust_lambda']
-                )
+                pass
```

Comment:
Why is this the case?

Reply:
This was debugging code that I should have removed.  I made the fix.

Reply:
This was taken care of by https://github.com/uber/ludwig/pull/646/commits/0b4fb5db897ce7a3e234897924664751d513af72